orchagent/lagid.h

```diff
@@ -0,0 +1,44 @@
+#ifndef SWSS_LAGID_H
+#define SWSS_LAGID_H
+
+#include "dbconnector.h"
+#include "sal.h"
+#include "schema.h"
+#include "redisapi.h"
+
+using namespace swss;
+using namespace std;
+
+#define LAG_ID_ALLOCATOR_ERROR_DELETE_ENTRY_NOT_FOUND  0
```

Comment:
Not finding these error codes being used, can be used in lagids.lua ?

Reply:
Yes. These errors are used in lagids.lua as hardcode. Here I have just defined these in order to indicate that these error numbers are taken to avoid conflict with any new error definition in the future